sql/updates/world/master/2017_12_05_20756_world.sql

```diff
@@ -0,0 +1,262 @@
+UPDATE `creature_template` SET `ScriptName` = 'npc_starry_sky' WHERE `entry` = 39681;
+UPDATE `creature_template` SET `ScriptName` = 'npc_isiset_mirror_image' WHERE `entry` IN (39720, 39721, 39722);
+UPDATE `creature_template` SET `ScriptName` = 'npc_isiset_spatial_flux', `InhabitType` = 12, `unit_flags` = 33554496 WHERE `entry` = 48707;
+UPDATE `creature_template` SET `ScriptName` = 'npc_isiset_energy_flux', `unit_flags` = 33554496, `speed_walk` = 5.5/2.5, `speed_run` = 5.5/2.5 WHERE `entry` = 48709;
+UPDATE `creature_template` SET `ScriptName` = 'npc_hoo_spatial_flux', `InhabitType` = 12, `unit_flags` = 33554496 WHERE `entry` = 39612;
+UPDATE `creature_template` SET `ScriptName` = 'npc_hoo_energy_flux', `unit_flags` = 33554496, `speed_walk` = 5.5/2.5, `speed_run` = 5.5/2.5 WHERE `entry` = 44015;
+
+-- Brann Bronzebeard more text
+DELETE FROM `creature_text` WHERE `CreatureID` = 39908 AND `GroupID` IN (13, 14);
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(39908, 13, 0, 'Blasted titans... Why do they use a different set of mechanisms at each of their installations?', 12, 0, 100, 432, 0, 0, 39940, 0, 'Brann Bronzebeard - idle'),
+(39908, 14, 0, 'This symbol, I think I''ve seen this before...', 12, 0, 100, 432, 0, 0, 40104, 0, 'Brann Bronzebeard - idle'); 
+
+-- Add missing npc text.
+DELETE FROM `npc_text` WHERE `ID` IN (15794);
+INSERT INTO `npc_text` (`ID`, `Probability0`, `Probability1`, `Probability2`, `Probability3`, `Probability4`, `Probability5`, `Probability6`, `Probability7`, `BroadcastTextID0`, `BroadcastTextID1`, `BroadcastTextID2`, `BroadcastTextID3`, `BroadcastTextID4`, `BroadcastTextID5`, `BroadcastTextID6`, `BroadcastTextID7`, `VerifiedBuild`) VALUES
+(15794, 1, 0, 0, 0, 0, 0, 0, 0, 40129, 0, 0, 0, 0, 0, 0, 0, -1);
+
+DELETE FROM `gossip_menu` WHERE (`MenuID` = 11339 AND `TextID` = 15794) OR (`MenuID` = 11348 AND `TextID` = 15815) OR (`MenuID` = 12512 AND `TextID` = 17600);
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`, `VerifiedBuild`) VALUES
+(11339, 15794, 0),
+(11348, 15815, 0),
+(12512, 17600, 0);
+
+DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_anhuur_activate_beacons'; -- rewritten script: spell_anhuur_activate_beacons -> spell_anhuur_handle_beacons
+DELETE FROM `spell_script_names` WHERE `ScriptName` IN ('spell_anhuur_handle_beacons,', 'spell_isiset_veil_of_sky', 'spell_isiset_supernova_filter', 'spell_isiset_mirror_image_starry_sky_spawner', 'spell_isiset_mirror_image_spawner', 'spell_isiset_image_explosion', 'spell_isiset_astral_rain_controller', 'spell_isiset_mana_shield_controller', 'spell_isiset_astral_familiar_controller', 'spell_isiset_call_of_sky', 'spell_hoo_energy_flux_target_selector', 'spell_isiset_energy_flux_target_selector', 'spell_hoo_arcane_energy_check');
```

Comment:
```sql
[...] WHERE `ScriptName` IN ('spell_anhuur_handle_beacons,', '
```
redundant comma at the end of the spell name, before the tick.
BTW, is there a (cosmetic/codestyle) standard for sorting the list below?

Reply:
No idea, i will reorganize into titled sections for easier reading